modules/fluid_structure_interaction/include/bcs/FluidFreeSurfaceBC.h

```diff
@@ -0,0 +1,34 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "IntegratedBC.h"
+
+class FluidFreeSurfaceBC;
+
+template <>
+InputParameters validParams<FluidFreeSurfaceBC>();
```

Comment:
We should not add the legacy parameter stuff to new code IMO. @aeslaughter ?

Reply:
And this is the only object where you're doing that. So I suggest to just remove this.

Reply:
along with the forward declaration above, which won't be needed anymore